travis does not fail with headers issues

I've also encountered what @alcides describes on #515 about the missing headers.

it seems that Travis does not catch this.

After all the tests have run:

Tests run: 539, Failures: 0, Errors: 0, Skipped: 0


[INFO]

[INFO] --- maven-antrun-plugin:1.7:run (default) @ spoon-core ---

[INFO] No ant target defined - SKIPPED

[INFO]

[INFO] --- animal-sniffer-maven-plugin:1.14:check
(ensure-java-1.6-class-library) @ spoon-core ---

[INFO] Checking unresolved references to
org.codehaus.mojo.signature:java16:1.0

[INFO]

[INFO] --- maven-jar-plugin:2.5:jar (default-jar) @ spoon-core ---

[INFO] Building jar:
/Users/alcides/Code/Research/spoon/target/spoon-core-5.1.0-SNAPSHOT.jar

[INFO]

[INFO] --- maven-dependency-plugin:2.8:copy-dependencies
(copy-dependencies) @ spoon-core ---

[INFO] junit-4.11.jar already exists in destination.

[INFO] system-rules-1.9.0.jar already exists in destination.

[INFO] jsap-2.1.jar already exists in destination.

[INFO] log4j-1.2.17.jar already exists in destination.

[INFO] commons-io-2.4.jar already exists in destination.

[INFO] org.eclipse.jdt.core-3.12.0.v20150913-1717.jar already exists in
destination.

[INFO] hamcrest-core-1.3.jar already exists in destination.

[INFO] mockito-all-2.0.2-beta.jar already exists in destination.

[INFO] ant-1.6.5.jar already exists in destination.

[INFO]

[INFO] --- maven-assembly-plugin:2.4:attached (make-assembly) @ spoon-core
---

[INFO] Building jar:
/Users/alcides/Code/Research/spoon/target/spoon-core-5.1.0-SNAPSHOT-jar-with-dependencies.jar

[INFO]

[INFO] --- maven-checkstyle-plugin:2.17:checkstyle (default) @ spoon-core
---

[INFO] Starting audit...

Audit done.

[WARNING] Unable to locate Source XRef to link to - DISABLED

[INFO]

[INFO] --- license-maven-plugin:2.11:check (default) @ spoon-core ---

[INFO] Checking licenses...

[WARNING] Missing header in:
/Users/alcides/Code/Research/spoon/bin/compiler/package.html

[WARNING] Missing header in: /Users/alcides/Code/Research/spoon/bin/pom.xml

[WARNING] Missing header in:
/Users/alcides/Code/Research/spoon/bin/LICENSE.txt

[WARNING] Missing header in:
/Users/alcides/Code/Research/spoon/bin/package.html

[WARNING] Missing header in:
/Users/alcides/Code/Research/spoon/bin/processing/package.html

[WARNING] Missing header in:
/Users/alcides/Code/Research/spoon/bin/reflect/eval/package.html

[WARNING] Missing header in:
/Users/alcides/Code/Research/spoon/bin/reflect/code/package.html

[WARNING] Missing header in:
/Users/alcides/Code/Research/spoon/bin/reflect/cu/package.html

[WARNING] Missing header in:
/Users/alcides/Code/Research/spoon/bin/BUILD.txt

[WARNING] Missing header in:
/Users/alcides/Code/Research/spoon/bin/reflect/declaration/package.html

[WARNING] Missing header in:
/Users/alcides/Code/Research/spoon/bin/reflect/factory/package.html

[WARNING] Missing header in:
/Users/alcides/Code/Research/spoon/bin/reflect/package.html

[WARNING] Missing header in:
/Users/alcides/Code/Research/spoon/bin/reflect/visitor/filter/package.html

[WARNING] Missing header in:
/Users/alcides/Code/Research/spoon/bin/reflect/reference/package.html

[WARNING] Missing header in:
/Users/alcides/Code/Research/spoon/bin/spooned-project-builder.xml

[WARNING] Missing header in:
/Users/alcides/Code/Research/spoon/bin/reflect/visitor/package.html

[WARNING] Missing header in:
/Users/alcides/Code/Research/spoon/bin/SpoonCleanUpPrefs.xml

[WARNING] Missing header in:
/Users/alcides/Code/Research/spoon/bin/SpoonFormatterPrefs.xml

[WARNING] Missing header in:
/Users/alcides/Code/Research/spoon/bin/src/main/assembly/project.xml

[WARNING] Missing header in:
/Users/alcides/Code/Research/spoon/bin/src/changes/changes.xml

[WARNING] Missing header in:
/Users/alcides/Code/Research/spoon/bin/src/main/resources/annotation.dtd

[WARNING] Missing header in:
/Users/alcides/Code/Research/spoon/bin/src/main/resources/spoon.properties

[WARNING] Missing header in:
/Users/alcides/Code/Research/spoon/bin/src/site/site.xml

[WARNING] Missing header in:
/Users/alcides/Code/Research/spoon/bin/src/main/resources/properties.dtd

[WARNING] Missing header in:
/Users/alcides/Code/Research/spoon/bin/src/test/resources/no-copy-resources/package.html

[WARNING] Missing header in:
/Users/alcides/Code/Research/spoon/bin/src/main/resources/spoonlet.dtd

[WARNING] Missing header in:
/Users/alcides/Code/Research/spoon/bin/src/test/resources/no-copy-resources/fr/inria/package.html

[WARNING] Missing header in:
/Users/alcides/Code/Research/spoon/bin/src/main/resources/log4j.properties

[WARNING] Missing header in:
/Users/alcides/Code/Research/spoon/bin/src/test/resources/no-copy-resources/fr/package.html

[WARNING] Missing header in:
/Users/alcides/Code/Research/spoon/bin/support/template/package.html

[WARNING] Missing header in:
/Users/alcides/Code/Research/spoon/bin/support/package.html

[WARNING] Missing header in:
/Users/alcides/Code/Research/spoon/bin/template/package.html

[WARNING] Missing header in:
/Users/alcides/Code/Research/spoon/bin/support/reflect/package.html

[INFO]
------------------------------------------------------------------------

[INFO] BUILD FAILURE

[INFO]
------------------------------------------------------------------------

[INFO] Total time: 02:26 min

[INFO] Finished at: 2016-02-02T12:28:16+00:00

[INFO] Final Memory: 36M/379M

[INFO]
------------------------------------------------------------------------

[ERROR] Failed to execute goal com.mycila:license-maven-plugin:2.11:check
(default) on project spoon-core: Some files do not have the expected
license header -> [Help 1]

[ERROR]

[ERROR] To see the full stack trace of the errors, re-run Maven with the -e
switch.

[ERROR] Re-run Maven using the -X switch to enable full debug logging.

[ERROR]

[ERROR] For more information about the errors and possible solutions,
please read the following articles:

[ERROR] [Help 1]
http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException


On Tue, Feb 2, 2016 at 2:14 PM, Gérard Paligot <notifications@github.com>
wrote:

> Do you talk about the license check? Because Travis check it. I need more
> information to know what is the problem.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/INRIA/spoon/issues/516#issuecomment-178592281>.
>

--- a/pom.xml
+++ b/pom.xml
@@ -547,6 +547,9 @@
                 <exclude>src/test/**</exclude>
                 <exclude>src/changes/**</exclude>
                 <exclude>src/site/**</exclude>
+                <exclude>target/**</exclude>
+                <exclude>bin/**</exclude>
+                <exclude>build/**</exclude>
               </excludes>
             </configuration>
             <executions>

